example/gmres/ex_real_A.cpp

```diff
@@ -117,7 +117,7 @@ int main(int argc, char* argv[]) {
   {
     // Read in a matrix Market file and use it to test the Kokkos Operator.
     KokkosSparse::CrsMatrix<ST, OT, EXSP> A =
-        KokkosKernels::Impl::read_kokkos_crst_matrix<
+        KokkosSparse::Impl::read_kokkos_crst_matrix<
```

Comment:
Since you're moving this into a different namespace anyway, maybe it should just be in ``KokkosSparse::``? I think it's pretty common enough for external users to read MatrixMarket files.

Reply:
Yeah, that is how it should be, I thought I had changed the namespace but maybe I forgot...
I should really re-import everything back and mark as deprecated but that will be pretty messy : (